arangod/IResearch/Search.cpp

```diff
@@ -264,6 +264,11 @@ std::string checkFieldsDifferentCollections(
 
 std::string check(SearchMeta const& search,
                   IResearchInvertedIndexMeta const& index) {
+#ifdef USE_ENTERPRISE
+  if (search.optimizeTopK != index._optimizeTopK) {
+    return "index optimize topK mismatches view optimize topK";
```

Comment:
NIT  to match actual parameter name.
```suggestion
    return "index optimizeTopK mismatches view optimizeTopK";
```

Reply:
I made it consistent with other messages